test/taoensso/faraday/tests/main.clj

```diff
@@ -132,75 +132,81 @@
 
 ;;;; Expressions support
 
-(let [i {:id 10 :name "update me"}]
+(let [i-wrong-name {:id 10 :name "update me"}
+      i-no-name {:id 10}
+      abc-set  {:id 10 :someset #{"a" "b" "c"}}
+      abcde-set {:id 10 :someset #{"a" "b" "c" "d" "e"}}
+      abcdef-set {:id 10 :someset #{"a" "b" "c" "d" "e" "f"}}]
   (after-setup!
    #(far/delete-item *client-opts* ttable {:id 10}))
 
   (expect ; Update expression support in update-item
    {:id 10 :name "foo"}
    (do
+     (far/delete-item *client-opts* ttable {:id 10})
+     (far/put-item *client-opts* ttable i-wrong-name)
      (far/update-item *client-opts* ttable
        {:id 10}
        {:update-expr     "SET #name = :name"
         :expr-attr-names {"#name" "name"}
         :expr-attr-vals  {":name" "foo"}
         :return          :all-new})))
+ 
+  (expect ; Update expression support in update-item check wierd capitalization
```

Comment:
What do you mean by "support for weird capitalization"?

There's a minor typo, btw. :)